docs/appnotes/0004-tams-for-data.md

```diff
@@ -0,0 +1,60 @@
+# When TAMS is a good fit for non-media data. And when it’s not
+
+## Abstract
+
+TAMS places few restrictions on the type of content stored within it.
+It has been developed primarily with audio and video in mind.
+But there are other types of timeline-based content one might wish to store in their media workflows.
+When is it appropriate to store these other types of content in TAMS?
+
+## Considerations
+
+### Indexing of content
+
+TAMS indexes content based on a small number of specific types of metadata.
+Content may be queried based on these types of metadata (e.g. Flow/Source ID, tags, time-ranges etc).
+These metadata apply to the Source/Flow as a whole.
+It is not possible to query the data itself, or metadata as it applies to a portion of the Source/Flow.
+As such, TAMS is usually not appropriate for the storing of timeline-based metadata.
```

Comment:
I can see what this is trying to say, but found it hard to parse until I'd read the whole document. How about this?

```suggestion
TAMS indexes Flows as a whole using a small number of pieces of metadata (Flow/Source ID, tags, etc.), however the only index that can be used inside a specific Flow is time.
It is not possible to query the data/content or metadata as it applies to a portion of the Source/Flow, which would be needed to query timeline-based metadata (e.g. it's not possible to query a transcript for occurrences of a specific word using TAMS).
As such, TAMS is usually not appropriate for the storing of timeline-based metadata.
However some examples of where it may be appropriate are given below.
```